### PR TITLE
ocr-numbers: Update ocr-numbers to reflect changes to canonical data

### LINKED ISSUE #999: ocr-numbers: check test version and update if necessary
As identified in #784, some of the tests currently don't reference the version number for the `canonical-data.json` that they are based on, `ocr-numbers` is one of them. In order to help with the ongoing maintenance of this track, we're in the process of checking the tests against the current version of the `canonical-data.json` - if they are up-to-date, then you can just add the version string, but if they are out-of-date, then the tests will need updating.

Tests should typically be ordered according to their order in the canonical data, since this makes it easier to check and update them in future. If you think that there's a better order, please let us know your reasoning in your PR.

If there are some additional tests that currently exist in this track but which aren't in the canonical data, these can be kept if you think that they are useful. They should be placed at the bottom of the test cases, and it would also be super-helpful if you could add a comment before them saying `# Additional tests for this track` to make it clear that they aren't part of the canonical data.

The version string should be placed just before the first test class and followed by a blank line. For this exercise, the version string is:
```
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

**Test suite:** [exercism/python/exercises/ocr-numbers/ocr_numbers_test.py](https://github.com/exercism/python/blob/master/exercises/ocr-numbers/ocr_numbers_test.py)
**Canonical data:** [exercism/problem-specifications/exercises/ocr-numbers/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/ocr-numbers/canonical-data.json)

### PR DESCRIPTION
**Pull request by emerali**
Resolves #999

### CODE REVIEW

**Comment by emerali** [comment]
Accidentally committed changes for another PR as well, will fix in a sec

File: exercises/ocr-numbers/example.py [hunk 1]
```diff
@@ -16,28 +16,25 @@ def split_ocr(ocr):
 OCR_LIST = [OCR_LIST[-1]] + OCR_LIST[:9]
 
 
-def number(ocr):
-    if (len(ocr) != ROW or len(ocr[0]) % COL or
-            any(len(r) != len(ocr[0]) for r in ocr)):
+def convert(input_grid):
```
Comment (m-a-ge) [inline exercises/ocr-numbers/example.py]: Exercise placeholder should be also updated in this case:
https://github.com/exercism/python/blob/master/exercises/ocr-numbers/ocr_numbers.py

**Comment by emerali** [comment]
@m-a-ge I've made the requested change.

**Comment by m-a-ge** [comment]
Merged, thanks @emerali!!

**Pull request merged**